libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/RouteStyleDescriptor.kt

```diff
@@ -4,12 +4,12 @@ package com.mapbox.navigation.ui.route
  * This class is used for describing the route line color(s) at runtime.
  *
  * @param routeIdentifier a string that identifies routes which should have their color overridden
- * @param lineColorResourceId the color of the route line
- * @param lineShieldColorResourceId the color of the shield line which appears below the route line
+ * @param lineColor the color of the route line
+ * @param lineCasingColor the color of the casing line which appears below the route line
  * and is normally wider providing a visual border for the route line.
  */
 data class RouteStyleDescriptor(
     val routeIdentifier: String,
-    val lineColorResourceId: Int,
-    val lineShieldColorResourceId: Int
+    val lineColor: Int,
+    val lineCasingColor: Int
```

Comment:
This is breaking SemVer. Is that intentional?

Reply:
```
src/main/java/com/******/navigation/ui/route/RouteStyleDescriptor.kt:13: error: Attempted to change parameter name from lineColorResourceId to lineColor in constructor com.******.navigation.ui.route.RouteStyleDescriptor [ParameterNameChange]
src/main/java/com/******/navigation/ui/route/RouteStyleDescriptor.kt:14: error: Attempted to change parameter name from lineShieldColorResourceId to lineCasingColor in constructor com.******.navigation.ui.route.RouteStyleDescriptor [ParameterNameChange]
src/main/java/com/******/navigation/ui/route/RouteStyleDescriptor.kt:1: error: Attempted to change parameter name from lineColorResourceId to lineColor in method com.******.navigation.ui.route.RouteStyleDescriptor.copy [ParameterNameChange]
src/main/java/com/******/navigation/ui/route/RouteStyleDescriptor.kt:1: error: Attempted to change parameter name from lineShieldColorResourceId to lineCasingColor in method com.******.navigation.ui.route.RouteStyleDescriptor.copy [ParameterNameChange]
src/main/java/com/******/navigation/ui/route/RouteStyleDescriptor.kt:14: error: Added method com.******.navigation.ui.route.RouteStyleDescriptor.getLineCasingColor() [AddedMethod]
src/main/java/com/******/navigation/ui/route/RouteStyleDescriptor.kt:13: error: Added method com.******.navigation.ui.route.RouteStyleDescriptor.getLineColor() [AddedMethod]
/root/code/libnavigation-ui/api/current.txt:632: error: Removed method com.******.navigation.ui.route.RouteStyleDescriptor.getLineColorResourceId() [RemovedMethod]
/root/code/libnavigation-ui/api/current.txt:633: error: Removed method com.******.navigation.ui.route.RouteStyleDescriptor.getLineShieldColorResourceId() [RemovedMethod]
Aborting: Found compatibility problems checking the public API (/root/code/libnavigation-ui/src/main/java) against the API in /root/code/libnavigation-ui/api/current.txt

> Task :libnavigation-ui:checkApi FAILED
```

Reply:
I didn't realize it would break semver.  A customer suggested the change and it seemed harmless.